app/commands/donations/subscription/activate.rb

```diff
@@ -0,0 +1,7 @@
+class Donations::Subscription::Activate
+  include Mandate
+
+  initialize_with :subscription
+
+  def call = subscription.update!(status: :active)
```

Comment:
This command is currently a bit silly. I felt like this might be warranted still as it is called from multiple places, but 🤷